src/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/ValueToItems.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tools.BootstrapTasks
+{
+	public class ValueToItems : Task
```

Comment:
Another useful input might `AdditionalMetadata` which takes a list of key-value pairs to add to all the generated items. Like `CreateItem` task ( https://msdn.microsoft.com/en-us/library/s2y3e43x.aspx )